app/scripts/react-directives/device-manager/config-editor/configEditorPageStore.js

```
@@ -220,6 +220,13 @@ function getTopics(portTabs, deviceTypesStore) {
   return topics;
 }
 
+function getDeviceTypeFromConfig(deviceConfig) {
+  if (deviceConfig?.device_type) {
+    return deviceConfig.device_type;
+  }
+  return 'protocol:' + (deviceConfig?.protocol || 'modbus');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)